lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1445,8 +1445,8 @@ def jsonrpc_claim_show(self, name, txid=None, nout=None):
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
     def jsonrpc_get(
-            self, name, file_name=None, stream_info=None, timeout=None,
-            download_directory=None, wait_for_write=True):
+    self, name, file_name=None, stream_info=None, timeout=None,
```

Comment:
not sure what this one means. its bad style to have a line break after parentheses? whats the right way to format this?